source/extensions/filters/http/jwt_authn/verifier.cc

```diff
@@ -48,11 +48,30 @@ class ContextImpl : public Verifier::Context {
   // Stores an authenticator object for this request.
   void storeAuth(AuthenticatorPtr&& auth) { auths_.emplace_back(std::move(auth)); }
 
+  // Add a pair of (issuer, payload), called by Authenticator
+  void addPayload(const std::string& issuer, const ProtobufWkt::Struct& payload) {
+    payload_pairs_.push_back({issuer, payload});
```

Comment:
Can `payload_pairs_` just be a struct and you insert to it here. So setPayload doesn't have to iterate and copy them once again?

Reply:
Done